spmd/tensor/parallel/__init__.py

```diff
@@ -5,8 +5,8 @@
 
 from spmd.tensor.parallel.style import (
     ColwiseParallel,
-    ParallelStyle,
     PairwiseParallel,
+    ParallelStyle,
```

Comment:
reorder

Reply:
Opps, maybe I didn't update my local change... NVM. Thanks for the fix.